### PR TITLE
add support for bulk_create

### PR DESCRIPTION
**Pull request by Mng-dev-ai**

### CODE REVIEW

**Review by aminalaee — COMMENTED** [review]
Thanks for this.

Just a few notes, the `database.execute_many` is actually running `execute` in a loop in `databases`. I think that needs to be mentioned here, or if you like a PR for that on `databases` would be welcome (I have an unfinished PR for that which can give you an idea) :)

And also do you think the validation part can be shared with the `create` so we don't duplicate that? I know it's not a big deal, probably 5 lines.

**Comment by Mng-dev-ai** [comment]
Yes, you're right I made some changes.

File: orm/models.py [hunk 1]
```diff
@@ -423,6 +424,15 @@ async def create(self, **kwargs):
 
         return instance
 
+    async def bulk_create(self, objs: typing.List[typing.Dict]) -> None:
+        new_objs = []
+        for obj in objs:
+            obj = self._validate_kwargs(**obj)
+            new_objs.append(obj)
```
Comment (aminalaee) [inline orm/models.py:431]:
```suggestion
        new_objs = [self._validate_kwargs(**obj) for obj in objs]
```

**Review by aminalaee — COMMENTED** [review]
I think only documentation is needed, this looks good now.

**Review by aminalaee — APPROVED** [review]
Thank you for this

**Pull request merged**